ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/ClipGeometry.qml

```diff
@@ -0,0 +1,198 @@
+        }
+      }
+      targetScale: 15151632
+    }
+
+    // Create an envelope outside of Colorado
+    Envelope {
+      id: outsideEnvelope
+      xMax: -11858344.321294
+      xMin: -12201990.219681
+      yMax: 5297071.577304
+      yMin: 5147942.225174
+      spatialReference: SpatialReference {
+        wkid: 3857
+      }
+    }
+
+    // Create an envelope intersecting Colorado
+    Envelope {
+      id: intersectingEnvelope
+      xMax: -11962086.479298
+      xMin: -12260345.183558
+      yMax: 4566553.881363
+      yMin: 4332053.378376
+      spatialReference: SpatialReference {
+        wkid: 3857
+      }
+    }
+
+    // Create an envelope inside of Colorado
+    Envelope {
+      id: containedEnvelope
+      xMax: -11431488.567009
+      xMin: -11655182.595204
+      yMax: 4741618.772994
+      yMin: 4593570.068343
+      spatialReference: SpatialReference {
+        wkid: 3857
+      }
+    }
+
+    // Create a graphics overlay for the Colorado geometry
+    GraphicsOverlay {
+      id: coloradoOverlay
+
+      // Colorado
+      Graphic {
+        id: coloradoGraphic
+        geometry: colorado
+        symbol: coloradoFillSymbol
+      }
+    }
+
+    // Create a graphics overlay to contain the clipping envelopes
+    GraphicsOverlay {
+      id: envelopesOverlay
+
+      // Outside envelope
+      Graphic {
+        id: outsideEnvelopeGraphic
+        geometry: outsideEnvelope
+        symbol: redOutline
+      }
+
+      // Intersecting envelope
+      Graphic {
+        id: intersectingEnvelopeGraphic
+        geometry: intersectingEnvelope
+        symbol: redOutline
+      }
+
+      // Contained envelope
+      Graphic {
+        id: containedEnvelopeGraphic
+        geometry: containedEnvelope
+        symbol: redOutline
+      }
+    }
+
+    // Create a graphics overlay for the clipped graphics
+    GraphicsOverlay {
+      id: clippedAreasOverlay
+    }
+  }
+
+  // Create a button that clips the geometry into the envelopes
+  Button {
+    id: clipButton
+    anchors {
+      horizontalCenter: parent.horizontalCenter
+      bottom: parent.bottom
+      bottomMargin: 25 * scaleFactor
+    }
+    text: "Clip"
+    onClicked: {
+      // Immediately hide the Colorado graphic to prevent overlap
+      coloradoOverlay.visible = false;
+
+      // Iterate through the clipping envelopes
+      envelopesOverlay.graphics.forEach(function(graphic) {
+
+        // Create a variable that contains the clip result, which is an envelope of the overlap between colorado and the current graphic iteration
+        var clippedGeometry = GeometryEngine.clip(coloradoGraphic.geometry, graphic.geometry.extent);
+
+        // Check if null
+        if (clippedGeometry !== null) {
+
+          // Create a new graphic using the clip envelope, and fill it in with the colorado fill symbol
+          var clippedArea = ArcGISRuntimeEnvironment.createObject("Graphic", { geometry: clippedGeometry, symbol: coloradoFillSymbol });
```

Comment:
Rename to `clippedGraphic`. Area may confuse people into thinking this is a geometry object.